tests/test_resources/test_clusters/test_cluster.py

```diff
@@ -847,8 +849,10 @@ def test_default_env_var_run(self, cluster):
     def test_cluster_run_within_cluster(self, cluster):
         remote_run = rh.function(run_in_no_env).to(cluster)
         res = remote_run("echo hello")
+        exp = cluster.run("echo hello")
+
         assert res[0][0] == 0
-        assert res[0][1] == "hello\n"
+        assert res[0][1] == exp[0][1]
```

Comment:
@rohinb2 should we just be testing that running remote is the same as from within the cluster, or also the specific output? tho currently they both return `"hello\n\n"`, which we should probably address